lib/StaticAnalyzer/Checkers/ReturnValueCheckStatisticsCollector.cpp

```diff
@@ -0,0 +1,114 @@
+//==- ReturnValueCheckStatisticsCollector.cpp --------------------*- C++ -*-==//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
Could you please add a comment describing what statistics this checker is collecting, what is the intended use, what are the input config variables.